scripts/batting_average.sql

```diff
@@ -0,0 +1,88 @@
+# Batting Average = Total hits / total at-bats
+
+# Make a table of players' total hits and at bats each game
+# Join stats, we'll need it later
+DROP TABLE IF EXISTS player_game_stats;
+
+CREATE TABLE player_game_stats AS
+	SELECT 
+		bc.batter,
+		bc.game_id, 
+		date(g.local_date) AS local_date, 
+		bc.atBat, 
+		bc.Hit
+	FROM batter_counts bc
+	JOIN game g ON bc.game_id=g.game_id
+	WHERE bc.atBat > 0
+;
```

Comment:
Putting the where bc.atBat>0 did not fix the division by 0 error for me on MariaDB. See line 26 comment for possible solution.